crates/ruff_dev/Cargo.toml

```diff
@@ -32,6 +33,7 @@ imara-diff = { workspace = true }
 indicatif = { workspace = true }
 itertools = { workspace = true }
 libcst = { workspace = true }
+markdown = { version = "1.0.0" }
```

Comment:
this is a bit unfortunate but implementing a markdown to HTML converter is a bit out of scope right now ;)